Library/Formula/llvm.rb

```diff
@@ -198,12 +196,6 @@ def install
 
     args << "-DLLVM_ENABLE_RTTI=On" if build.with? "rtti"
 
-    if build.with? "assertions"
-      args << "-DLLVM_ENABLE_ASSERTIONS=On"
-    else
-      args << "-DCMAKE_CXX_FLAGS_RELEASE='-DNDEBUG'"
```

Comment:
Not that this doesn't need that flag; that just seemed like a more general solution, because I'd guess that this `-DNDEBUG` thing would happen with most of the CMake formulas